_posts/newsletters/2024-01-24-newsletter.md

```diff
@@ -0,0 +1,382 @@
+---
+layout: post
+title: US-RSE January 2024 Newsletter
+subtitle: January 2024 
+category: newsletter
+tags: [newsletter, January]
+---
+
+In this monthly newsletter, we share recent, current, and planned activities of the US-RSE Association, and related news that we think is of interest to US-RSE members. Newsletters are also available on our [website](https://us-rse.org/newsletters/) alongside the growing resources and information on the US-RSE Association. To receive our newsletter, [join US-RSE](https://us-rse.org/join/).
+
+In this issue:
+* [Two-year Alfred P. Sloan Foundation Grant](#grant-announcement)
+* [US-RSE Conference 2024](#conference2024)
+* [Steering Committee Elections](#sc-elections)
+* [Community News](#news)
+    * [Funders Talk Series](#funder-series)
+    * [Member Spotlight](#member-spotlight)
+    * [Member Snapshot](#member-snapshot)
+    * [RSE Group Leaders' Network](#rse-gln)
+    * [DEI Working Group](#dei-working-group)
+    * [User Experience Working Group](#wg-ux)
+    * [Community Calls](#community-calls)
+    * [Community Manager office hours](#community-manager)
+* [Interesting Events and Opportunities](#events)
+* [Interesting Reads, Videos, or Podcasts](#reads)
+* [US-RSE Outreach](#outreach)
+* [US-RSE Financials](#financials)
+* [Get Involved](#involved)
+* [Recent Job Postings](#jobs)
+
+
+
+# **Two-year Alfred P. Sloan Foundation Grant**
+
+The [generous grant](https://us-rse.org/2023-04-27-sloan-grant-initiatives/) by the Alfred P. Sloan Foundation  will provide support in three important areas: the recruitment of dedicated staff, the development of activities and initiatives, and the enhancement of organizational tools and community well-being. 
+
+* A large part of US-RSE’s success has been due to the dedication and hard work of volunteers. While we don’t want to see the community-driven nature change, we recognize that relying on pure volunteer effort is limiting. For this reason the two new staff positions of a full-time Community Manager and a part-time Executive Director were created. We are excited that Sophia Anyatonwu started on December 11, 2023 as our Community Manager and Sandra Gesing started on November 1, 2023 as our Executive Director.
+* Three main activities and initiatives will be funded through the grant: The US-RSE Annual Conference, a Community Funds Program, and an Awards Program.
+* Developing and maintaining a vibrant and functional community network requires infrastructure. To alleviate the struggles of the current process, we’ll invest in infrastructure for managing contact information for those in the network, facilitating networking opportunities between members, allowing people to choose their level of involvement in network activities, and communicating with members. 
+
+## Community and Travel Funds program: Joint Maths Meetings 2024
```

Comment:
"Maths"? :smile:

Reply:
it's the UK version of "Math" :)

Reply:
I know, I just get a kick out of it every time I see it.

Reply:
I have a t-shirt that say "do the maths" on the back...

Reply:
I will change to Mathematics :)

Reply:
Thanks! We fixed/changed in latest commit!